File: Rock Paper Scissors with CPP/export/cppsrc/Strings.cpp
#pragma once

#include <cstring>
#include <string>
#include <regex>

namespace Strings
{
    auto stringSize(auto text)
    {
        return static_cast<int>(strlen(text));
    }

    auto stringReplace(auto text, const auto toReplace, const auto replacement)
    {
        std::string str = text;
        return std::regex_replace(str, std::regex(toReplace), replacement);
    }

    auto stringSub(auto text, auto startInt, auto endInt)
    {
        std::string str = text;
        return str.substr(startInt, endInt);
    }
}

File: Target Checker/export/jssrc/Strings.js
module.exports = class Strings
{
    static stringSize(text) 
    {
        return text.length;
    }

    static stringReplace(text, toReplace, replacement)
    {
        return text.replace(toReplace, replacement);
    }

    static stringSub(text, startInt, endInt)
    {
        return text.substr(startInt, endInt);
    }
}